css/style.css

```diff
@@ -5,6 +5,13 @@ body {
   margin: 0;
 }
 
+.desk-item,
+.desk-item1,
+.flex-item2-desk,
+.technology-desk {
+  display: none;
+}
+
```

Comment:
- [ ] [REQUIRED] Kindly ensure that in the about section, the left section has a width of 40%, and the right section takes a width of 60% to match the original design

Reply:
> 
If you don't mind I need you to elaborate me the relationship between the issue you raised and the code you hover over or highlighted. I just want to understand it well.

Reply:
Most of the code in this PR is not highlightable since most of it is unchanged. Only a short selection of it is the one that has changes, just ignore the relationship here